src/processes/follow-topic-process.js

```diff
@@ -140,10 +183,12 @@ const followTopicProcess = async (job, done) => {
         const channelState = await chat.watch();
         // console.log('::: channelState.messages :::', channelState);
         if (channelState.messages.length === 0) {
-          await chat.sendMessage({
-            text: communityMessageFormat.message,
-            user_id: communityMessageFormat.user_id,
-            is_auto_message: true
+          await sendSignedMessage({
+            serverChannel: chat,
+            targetUserId: data?.user_id,
+            senderUsername: senderUser?.username,
+            communityMessageFormatUserId: communityMessageFormat.user_id,
+            text: communityMessageFormat?.message
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `sendSignedMessage` function is used correctly here to send a signed message if there are no messages in the channel state.
```diff
+ await sendSignedMessage({
+   serverChannel: chat,
+   targetUserId: data?.user_id,
+   senderUsername: senderUser?.username,
+   communityMessageFormatUserId: communityMessageFormat.user_id,
+   text: communityMessageFormat?.message
+ });
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->